src/index.ts

```diff
@@ -64,6 +67,8 @@ process.on('unhandledRejection', err => {
     registerMetricsExporter(projectId);
     await server.initialize();
     await server.start();
+
+    console.info('✅ Server started on port ' + port);
```

Comment:
Jeg liker det når hele adressen skrives ut fordi da kan jeg cmd+click den for å åpne fra terminal. Forslag:
```suggestion
    console.info('✅ Server started at http://localhost:' + port);
```

Reply:
Lurt 👍